homiebot-web.csproj

```diff
@@ -1,32 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>homiebot_web</RootNamespace>
   </PropertyGroup>
-
   <ItemGroup>
-    <PackageReference Include="AWSSDK.S3" Version="3.5.7.4" />
-    <PackageReference Include="DSharpPlus" Version="4.0.0" />
-    <PackageReference Include="DSharpPlus.CommandsNext" Version="4.0.0" />
-    <PackageReference Include="DSharpPlus.VoiceNext" Version="4.0.0" />
-    <PackageReference Include="DSharpPlus.VoiceNext.Natives" Version="1.0.0" />
-    <PackageReference Include="FileContextCore" Version="3.4.0" />
-    <PackageReference Include="Google.Cloud.TextToSpeech.V1" Version="2.1.0" />
-    <PackageReference Include="Magick.NET-Q8-AnyCPU" Version="7.22.2.2" />
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.15.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.23" />
-    <PackageReference Include="Microsoft.CognitiveServices.Speech" Version="1.14.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.7" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Cosmos" Version="3.1.7" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="NAudio" Version="1.10.0" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="1.0.3" />
-    <PackageReference Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta13" />
-    <PackageReference Include="System.Net.Http.Json" Version="5.0.0" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.1.13"/>
+    <PackageReference Include="DSharpPlus" Version="4.0.1"/>
+    <PackageReference Include="DSharpPlus.CommandsNext" Version="4.0.1"/>
+    <PackageReference Include="DSharpPlus.VoiceNext" Version="4.0.1"/>
+    <PackageReference Include="DSharpPlus.VoiceNext.Natives" Version="1.0.0"/>
+    <PackageReference Include="FileContextCore" Version="3.4.0"/>
+    <PackageReference Include="Google.Cloud.TextToSpeech.V1" Version="2.2.0"/>
+    <PackageReference Include="Magick.NET-Q8-AnyCPU" Version="8.0.1"/>
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.20.1"/>
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.27"/>
+    <PackageReference Include="Microsoft.CognitiveServices.Speech" Version="1.17.0"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.7"/>
```

Comment:
This and the line below it need to be reverted for the offline brain to keep working